.eslintrc.json

```diff
@@ -9,10 +9,10 @@
     "eslint:recommended",
     "plugin:flowtype/recommended",
     "plugin:react/recommended",
-    "plugin:jest/recommended",
-    "prettier",
-    "prettier/flowtype",
-    "prettier/react"
+    "plugin:jest/recommended"
```

Comment:
please do not change this configurations files if not in a dedicated PR with a clear reason. rules are usually there for a reason

Reply:
Sorry, not intentional